src/routes.js

```diff
@@ -3,6 +3,24 @@
 var routes = {},
     util = require('util');
 
+var networkTexts = {
```

Comment:
Any chance we could put texts like these in a separate file and `require` them? For example `src/texts.js`.